spec/support/features/user_login.rb

```diff
@@ -0,0 +1,9 @@
+def login_user(email, password)
```

Comment:
Unused method argument - `email`. If it's necessary, use `_` or `_email` as an argument name to indicate that it won't be used. You can also write as `login_user(*)` if you want the method to accept any arguments but don't care about them.<br>Unused method argument - `password`. If it's necessary, use `_` or `_password` as an argument name to indicate that it won't be used. You can also write as `login_user(*)` if you want the method to accept any arguments but don't care about them.